p2p/protocol/circuitv2/util/pbconv.go

```diff
@@ -36,13 +36,10 @@ func PeerToPeerInfoV1(p *pbv1.CircuitRelay_Peer) (peer.AddrInfo, error) {
 }
 
 func PeerInfoToPeerV1(pi peer.AddrInfo) *pbv1.CircuitRelay_Peer {
-	var addrs [][]byte
-	if len(pi.Addrs) > 0 {
-		addrs = make([][]byte, 0, len(pi.Addrs))
-	}
+	addrs := make([][]byte, len(pi.Addrs))
```

Comment:
Let’s keep the old behaviour of having this be nil. In this case I don’t think this matters, but generally we can’t be sure if callers check this.